EKF/estimator_interface.cpp

```diff
@@ -478,9 +478,21 @@ void EstimatorInterface::setDragData(const imuSample &imu)
 bool EstimatorInterface::initialise_interface(uint64_t timestamp)
 {
 	// find the maximum time delay the buffers are required to handle
-	float max_time_delay_ms = math::max(_params.baro_delay_ms,
-					math::max(_params.auxvel_delay_ms,
-						_params.airspeed_delay_ms));
+	float max_time_delay_ms = 0.f;
+	// using Barometric altitude
+	if (??) {
```

Comment:
The default baro delay is 0ms and it will always be small so I don't think it's worth adding a check for it.

Reply:
@bresch - Done